crates/biome_diagnostics/src/diagnostic.rs

```diff
@@ -124,6 +132,35 @@ pub enum Severity {
     Fatal,
 }
 
+impl FromStr for Severity {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "hint" => Ok(Self::Information),
+            "info" => Ok(Self::Information),
+            "warn" => Ok(Self::Warning),
+            "error" => Ok(Self::Error),
+            v => Err(format!(
+                "Found unexpected value ({}), valid values are: info, warn, error.",
+                v
+            )),
+        }
+    }
+}
+
+impl Display for Severity {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::Hint => write!(f, "hint"),
```

Comment:
This should be `"info"` for now

Reply:
My thinking was that it's safe to parse this to `"hint"`, because it won't let users set the option to `"hint"` and if `Severity::Hint` is used internally and someone later wants to format/print it and `Hint` turns into `"info"`, they might go mad before looking at the `Display` implementation 😄 . But I see why you're saying it, I'm happy to change it.